src/test/java/org/cactoos/io/WriterAsOutputTest.java

```diff
@@ -52,11 +51,10 @@
 
     @Test
     public void writesLargeContentToFile() throws IOException {
-        final Path temp = this.folder.newFile("cactoos-1.txt-1")
+        final Path temp = this.folder
+            .newFile("cactoos-1.txt-1")
             .toPath();
-        try (final OutputStreamWriter writer = new OutputStreamWriter(
-            new FileOutputStream(temp.toFile()), StandardCharsets.UTF_8
-        )) {
+        try (Writer writer = Files.newBufferedWriter(temp)) {
```

Comment:
@ilyakharlamov please, check if is possible another kind of object instead of use a static method.